pkg/globalmanager/controllers/lifelonglearning/lifelonglearningjob.go

```diff
@@ -18,7 +18,9 @@ package lifelonglearning
 
 import (
 	"context"
+	"encoding/json"
 	"fmt"
+	"k8s.io/apimachinery/pkg/types"
```

Comment:
Follow the import order of go packages:
1. built-in packages
2. third-party packages
3. self-packages

Reply:
fix it